src/main/java/org/trivetia/mace/ae2/spatial/SpatialIO.java

```diff
@@ -0,0 +1,51 @@
+package org.trivetia.mace.ae2.spatial;
+
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.registry.RegistryNamespaced;
+import net.minecraftforge.fml.common.event.FMLInterModComms;
+import org.trivetia.mace.config.ae2.ConfigAe2;
+
+public class SpatialIO 
+{    
+        
+    private final List<String> modsToAdd = new ArrayList<>(); //Contains Mods which will be loaded

```

Comment:
A space between the // and the comment would be nice (fancy stuff), you don't have to do this.